### DISCUSSION TITLE
Unauthorized error when installing with custom nuget library

### DISCUSSION THREAD

**Discussion by mbakervti** [dotnet/docfx#9579] [General]
I had a hard time installing DocFx and I thought I would put the solution I found here just in case either there's something that can be done about it, or other's might be searching for the same issue. I was using the command in a command window to install global

`dotnet tool install --global docfx` 

I kept getting an error message that something was 401 (Unauthorized).

```
---> System.Net.Http.HttpRequestException: Response status code does not indicate success: 401 (Unauthorized).
   at System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode()
   at NuGet.Protocol.HttpSource.<>c__DisplayClass15_0`1.<<GetAsync>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at NuGet.Common.ConcurrencyUtilities.ExecuteWithFileLockedAsync[T](String filePath, Func`2 action, CancellationToken token)
   at NuGet.Common.ConcurrencyUtilities.ExecuteWithFileLockedAsync[T](String filePath, Func`2 action, CancellationToken token)
   at NuGet.Protocol.HttpSource.GetAsync[T](HttpSourceCachedRequest request, Func`2 processAsync, ILogger log, CancellationToken token)
 -- MORE LINES . . .
```
I rebooted and tried it again and then I got an additional line in the error that I hadn't seen before:
`Unhandled exception: NuGet.Protocol.Core.Types.FatalProtocolException: Unable to load the service index for source https://pkgs.dev.azure.com/[PATH TO COMPANY NUGET]/nuget/v3/index.json.`
I have no idea why I didn't see this before rebooting and trying again.
Our company has some NuGet packages that we build and maintain for internal use. To fix this I opened VS 2022, opened one of the solutions, opened the NuGet package UI, and un-checked the NuGet package location for the company...

![image](https://github.com/dotnet/docfx/assets/82122331/259ececf-90cb-4e70-bba8-c4def83a90ae)

After clearing that checkbox I closed VS 2022 and ran the install again  `dotnet tool install --global docfx` It worked this time.
Hope this might help someone.
Mike

**Comment by AmbichlH** [comment]
Thank you. A life saver :)
